app/views/about/index.html.erb

```diff
@@ -0,0 +1,91 @@
+<% @page_title = t('blacklight.about.page_title', :application_name => application_name) %>
+
+<%= render 'shared/start_over_row' %>
+
+<div id="content" class="col-lg-12">
+<h1 class='page-heading'><%= t('blacklight.about.title') %></h1>
+<p>
+  The Princeton University Library’s Digital Maps and Geospatial Data website
+  allows researchers to discover and access online maps and geospatial data in
+  the Princeton Maps Collection, one of the largest cartographic collections
+  at any academic institution.
+</p>
+<h2>Types of Maps found in the Portal</h2>
+
+<p>Please note that this Portal represents only a subsection of the maps that
+  Princeton University Library has in its collection. [maybe add some text about how/where to
+  locate the maps that are not discoverable in this portal?]
+
+<h3>Sanborn Maps</h3>
+
+<p>The Sanborn Map Company is the oldest mapping company in the United States.
+  Founded in 1866, the company created detailed building maps of cities and towns
+  all across the country, primarily for use by insurance companies. These maps
+  also have been used extensively by local government agencies for over a century
+  in conducting their daily operations efficiently. Approximately 12,000 American
+  cities and towns are chronicled in the company's collection of over one million maps.
+  As a result of its history, the name "Sanborn" has become synonymous with this kind of map.
+  Genealogists, local history buffs, city planners, architects, and countless others have
+  been drawn to the quality of detail found in them. </p>
+
+<p>
+  <%=image_tag("about/pulmap-sanborn.jpg", alt: "Example of a Sandborn map of Princeton, Mercer County, New Jersey", width: "800px")%>
+</p>
+
+<h3>Scanned Maps</h3>
+<p>
+Scanned Maps are print maps that have been digitized and combined with geographic information, allowing for accurate positioning of the map on an interactive, modern globe. Many of the Scanned Maps available in the Digital Map and Geospatial Data Portal are historic, allowing researchers to compare past geographic boundaries and names with their modern counterparts. Princeton University Library has a strong commitment to scanning our print maps collection, so scanned maps are added to the portal on a regular basis.
```

Comment:
`and combined with geographic information, allowing for accurate positioning of the map on an interactive, modern globe` is misleading. I think it's fine to just say "digitized print maps".

Reply:
I was going for something that would tie these maps to GIS and Geospatial discovery. "Digitized print maps" that have no GIS information would not be included in this portal, or would they?